features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/vitals/JankStatsActivityLifecycleListener.kt

```diff
@@ -24,17 +32,45 @@ import java.util.concurrent.TimeUnit
 internal class JankStatsActivityLifecycleListener(
     private val vitalObserver: VitalObserver,
     private val internalLogger: InternalLogger,
-    private val jankStatsProvider: JankStatsProvider = JankStatsProvider.DEFAULT
+    private val jankStatsProvider: JankStatsProvider = JankStatsProvider.DEFAULT,
+    private var screenRefreshRate: Double = 60.0
 ) : ActivityLifecycleCallbacks, JankStats.OnFrameListener {
 
     internal val activeWindowsListener = WeakHashMap<Window, JankStats>()
     internal val activeActivities = WeakHashMap<Window, MutableList<WeakReference<Activity>>>()
+    internal var display: Display? = null
 
     // region ActivityLifecycleCallbacks
     @MainThread
     override fun onActivityCreated(activity: Activity, savedInstanceState: Bundle?) {
     }
 
+    @RequiresApi(Build.VERSION_CODES.N)
+    interface DDFrameMetricsListener : Window.OnFrameMetricsAvailableListener {
+        override fun onFrameMetricsAvailable(
+            window: Window,
+            frameMetrics: FrameMetrics,
+            dropCountSinceLastInvocation: Int
+        )
+
+        var frameDeadline: Long
```

Comment:
I believe this one can be removed

```suggestion
    interface DDFrameMetricsListener : Window.OnFrameMetricsAvailableListener {

        var frameDeadline: Long
```